rakelib/CompilationConfiguration.rb

```diff
@@ -45,6 +45,12 @@ def self.remove_swiftdoc()
       FileUtils.rm_rf(file)
     end
   end
+
+  def self.remove_swiftsourceinfo()
+    Dir.glob("#{MODULE_OUTPUT_PATH}/*/*.swiftsourceinfo").each do |file|
```

Comment:
Is that a new type of file since Xcode 11 or something? (Wondering if maybe we could avoid generating it in the first place by passing some special arg when we run `xcodebuild`?)

Reply:
Ok seems it's always generated no matter what: https://forums.swift.org/t/proposal-emitting-source-information-file-during-compilation/28794 so might as well keep the deletion code in here indeed.